rental-car-manager/README.md

```diff
@@ -10,6 +10,7 @@ To develop the extension, you must run both the Angular application and the edit
 In one terminal, for the Angular application:
```

Comment:
nit: move format changes into the commit that the original content was introduced in. (Here it should be pretty straightforward -- you can separate this commit into 2 and then squash the format-only commit into an earlier commit as needed?)